MREUnityRuntime/MREUnityRuntimeLib/Core/CollisionLayers.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MixedRealityExtension.Core
```

Comment:
The class and enum in this file are something that should be provided by the host app implementing MRE's.  All of these need docstrings for being exposed and used as public symbols in our runtime dll.